mne/evoked.py

```diff
@@ -825,16 +825,27 @@ def grand_average(all_evoked, interpolate_bads=True):
     return grand_average
 
 
-def _check_evokeds_ch_names_times(all_evoked):
+def _check_evokeds_ch_names_times(all_evoked, reorder=False):
     evoked = all_evoked[0]
     ch_names = evoked.ch_names
-    for ev in all_evoked[1:]:
+    for ii, ev in enumerate(all_evoked[1:]):
         if ev.ch_names != ch_names:
-            raise ValueError(
-                "%s and %s do not contain the same channels" % (evoked, ev))
+            if set(ev.ch_names) != set(ch_names):
+                raise ValueError(
+                    "%s and %s do not contain the same channels." % (evoked,
+                                                                     ev))
+            else:
+                if reorder is True:
```

Comment:
do you really need this extra nesting?

Reply:
It's unused. The rationale is that current behaviour is to not reorder, so I thought the old behaviour should somehow stay accessible.